PoterekM/django-react-todo
# Set up react frontend

## Description

This PR Creates the Frontend utilizing Create React App. 

The front end has fields for a title, description, and if the task is completed or not, which matches the fields in the backend. **These values are currently hardcoded until items can be fetched from the backend.** 

The interface allows for a user to:
- Add a task
- View Complete or incomplete tasks
- Edit or delete a task.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|   | :hammer: Refactoring       |
| | :100: Add tests            |
|  | :link: Update dependencies |
| | :scroll: Docs              |
## Related Issue

## After / New Behavior


![](https://cdn.zappy.app/72971564a86e11c0df04eefd3e750465.gif)

## Testing Steps / QA Criteria

1. Navigate to frontend directory `cd frontend`
2. start server `npm start`
3. Navigate to http://localhost:3000/

You should be able to:
- Click a button to add a task
- toggle between "complete" or "incomplete" tasks
- select edit and delete buttons

## TODOs
- [ ] Set up Axios
- [ ] Remove statically typed data to gather information from Django
